metric_metadata/metadata_map/map.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2015 Square Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata_map
```

Comment:
`metadata_map` is a feature-complete, but inefficient, implementation for a `MetricMetadataAPI`.

It can be used for starting an MQE instance without any external dependencies (for demo purposes).